relationships/synthesis/INFRA-NR_K8S_CONTAINER-to-EXT-SERVICE.yml

```diff
@@ -0,0 +1,43 @@
+relationships:
+  - name: nrInfraK8sContainerEXTService
+    version: "1"
+    origins:
+      - OpenTelemetry
+    conditions:
+      - attribute: eventType
+        anyOf: [ "K8sContainerSample" ]
+      - attribute: entityType
+        anyOf: [ "CONTAINER" ]
+    relationship:
+      expires: P75M
+      relationshipType: HOSTS
+      source:
+        buildGuid:
+          account:
+            lookup: true
+          domain:
+            value: INFRA
+          type:
+            value: CONTAINER
+            valueInGuid: NA
+          identifier:
+            fragments:
+              - value: "k8s:"
+              - attribute: clusterName
+              - value: ":"
+              - attribute: namespaceName
+              - value: ":pod:"
+              - attribute: podName
+            hashAlgorithm: FARM_HASH
+      target:
+        buildGuid:
+          account:
+            attribute: accountId
+          domain:
+            value: EXT
+          type:
+            value: SERVICE
+          identifier:
+            fragments:
+              - attribute: service.name
```

Comment:
Just wondering - will the `service.name` on a container sample reflect the name of a related EXT-SERVICE entity ? 🤔

Reply:
Could it be the other way around, that EXT-SERVICE telemetry may tell us the container id ?